src/assets/user-info-settings/index.css

```diff
@@ -0,0 +1,255 @@
+html {
+  background: red;
+}
+
+body {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  width: 1280px;
+  margin: 0;
+  overflow: auto;
+  box-sizing: border-box;
+  background: #eaeaea;
+}
+button:active, .server-selection input:checked + label {
```

Comment:
I'm only gonna comment this the one time because it happens multiple times. For consistency sake, either all blocks should have no line breaks or they should all have line breaks. Some blocks are touching and some are not